docs/platforms/unreal/tracing/trace-propagation/dealing-with-cors-issues/index.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: Dealing with CORS Issues
```

Comment:
Is this relevant? CORS is a browser thing

Reply:
One can build an Unreal game for [HTML5](https://forums.unrealengine.com/t/ue5-export-to-html5/1625616/2) but considering that the corresponding engine's plugin is not supported since UE 4.24 it should be safe to remove this.